src/dropin.js

```diff
@@ -108,15 +112,35 @@ Dropin.prototype._initialize = function (callback) {
       this._emit('noPaymentMethodRequestable');
     }.bind(this));
 
-    this._mainView = new MainView({
-      client: this._client,
-      element: this._dropinWrapper,
-      model: this._model,
-      strings: strings
-    });
+    function createMainView() {
+      dropinInstance._mainView = new MainView({
+        client: dropinInstance._client,
+        element: dropinInstance._dropinWrapper,
+        model: dropinInstance._model,
+        strings: strings
+      });
+    }
+
+    paypalRequired = this._model.supportedPaymentOptions.indexOf(paymentOptionIDs.paypal) !== -1 || this._model.supportedPaymentOptions.indexOf(paymentOptionIDs.paypalCredit) !== -1;
```

Comment:
Any chance we'd want to do:

```js
Dropin.prototype._supportsPaymentOption = function (paymentOption) {
  return this._model.supportedPaymentOptions.indexOf(paymentOption) !== -1;
};
```

so that we could do: 

```js
paypalRequired = this._supportsPaymentOption(paymentOptionIDs.paypal) || this._supportsPaymentOption(paymentOptionIDs.paypalCredit)
```

I find that a little clearer, but am fine with the repetition if you feel strongly about it.